airflow/contrib/operators/sagemaker_deploy_operator.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.hooks.sagemaker_hook import SageMakerHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.exceptions import AirflowException
+
+
+class SageMakerDeployOperator(BaseOperator):
+
+    """
+       Deploy a SageMaker model to an endpoint
+       This operator returns the ARN of the endpoint created in Amazon SageMaker
+       :param config:
+       The configuration necessary to create a model, an endpoint-config and
+       an endpoint
+       :type config: dict
+       :param region_name: The AWS region_name
+       :type region_name: str
+       :param aws_conn_id: The AWS connection ID to use.
+       :type aws_conn_id: str
+       :param wait_for_completion: if the operator should block
+       until training job finishes
+       :type wait_for_completion: bool
+       :param check_interval: if wait is set to be true, this is the time interval
+       in seconds which the operator will check the status of the training job
+       :type check_interval: int
+       :param max_ingestion_time: if wait is set to be true, the operator will fail
+       if the training job hasn't finish within the max_ingestion_time in seconds
+       (Caution: be careful to set this parameters because training can take very long)
+       Setting it to None implies no timeout.
+       :type max_ingestion_time: int
+       :param operation: Whether to create an endpoint or update an endpoint. Must be
+       one of 'create' and 'update'.
+       :type operation: str
+       **Example**:
```

Comment:
Can you see how this renders in the docs:

`make -C docs html && open docs/_build/html/index.html`

(I guessed at the index filename from memory, so this might not be right.